README.md

```diff
@@ -9,10 +9,20 @@ Please view this template for guidance on creating issues: https://github.com/el
 ## Contributing to Elastic Security docs
 
 If you are an Elastic employee and would like to contribute to Elastic Security documentation: 
+
 1. Please clone and fork the `security-docs` repo. 
 2. Open an issue using the appropriate [template](https://github.com/elastic/security-docs/tree/master/.github/ISSUE_TEMPLATE).
 3. Check out the `main` branch and fetch the latest changes. 
 4. Check out a new branch and make your changes. 
 5. Save your changes and open a pull request. 
-6. Tag the the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting.  
+6. Tag the the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting. 
```

Comment:
I think this item also needs updating. I'm not sure what `tag` means here. Is it a mention in the Description field? or a specific label (set of labels?)